.github/workflows/eas.yml

```diff
@@ -147,6 +147,18 @@ jobs:
           slug: ${{ steps.preview-build.outputs.slug }}
           project-id: ${{ steps.preview-build.outputs.project-id }}
 
+      - name: 🧪 Run Android E2E Tests
+        working-directory: apps/${{ matrix.project }}
+        run: |
+          eas workflow:run .eas/workflows/android-e2e-test.yml \
+            --non-interactive \
+
+      - name: 🧪 Run iOS E2E Tests
+        working-directory: apps/${{ matrix.project }}
+        run: |
+          eas workflow:run .eas/workflows/ios-e2e-test.yml \
+            --non-interactive \
+
```

Comment:
Isn't it possible to specify ios / android tests in the workflow instead of having to do it here?

Reply:
combined